src/main/java/frc/robot/commands/Arm/SpinToArmAngle.java

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands.Arm;
+import frc.robot.subsystems.ArmSubsystem;
+import edu.wpi.first.wpilibj2.command.Command;
+
+public class SpinToArmAngle extends Command {
+
+    private double angle;
+
+  @SuppressWarnings({"PMD.UnusedPrivateField", "PMD.SingularField"})
+  private final ArmSubsystem armSubsystem;
+
+  public SpinToArmAngle(ArmSubsystem armSubsystem, double angle) {
+    this.armSubsystem = armSubsystem;
+    this.angle = angle;
+    addRequirements(armSubsystem);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    System.out.println("hello");
```

Comment:
Print statements like this should be removed if they are not helpful, or provide more context.
Such as
`"SpinToArmAngle: initialize.  angle=" + angle`